### PR TITLE
RFC: Clean up tableau definition

### PR DESCRIPTION
**Pull request by YingboMa**
This is pretty breaking. @ChrisRackauckas how should we update DiffEqDevTools and OrdinaryDiffEq?

### CODE REVIEW

**Comment by YingboMa** [comment]
The printing now looks like
```julia
julia> constructTsitouras5()
Tsitouras5{explicit, FSAL, stages=7, order=5, embedded order=4}:
A:
   ⋅                       ⋅                  ⋅                    ⋅                     ⋅                     ⋅                  ⋅
  0.161                    ⋅                  ⋅                    ⋅                     ⋅                     ⋅                  ⋅
 -0.008480655492356989    0.335480655492357   ⋅                    ⋅                     ⋅                     ⋅                  ⋅
  2.8971530571054935     -6.359448489975075  4.3622954328695815    ⋅                     ⋅                     ⋅                  ⋅
  5.325864828439257     -11.748883564062828  7.4955393428898365  -0.09249506636175525    ⋅                     ⋅                  ⋅
  5.86145544294642      -12.92096931784711   8.159367898576159   -0.071584973281401    -0.028269050394068383   ⋅                  ⋅
  0.09646076681806523     0.01               0.4798896504144996   1.379008574103742    -3.290069515436081     2.324710524099774   ⋅
b:
 0.09646076681806523  0.01  0.4798896504144996  1.379008574103742  -3.290069515436081  2.324710524099774  0.0
embedded b:
 0.09468075576583945  0.009183565540343254  0.4877705284247616  1.234297566930479  -2.7077123499835256  1.866628418170587  0.015151515151515152
c:
 0.0
 0.161
 0.327
 0.9
 0.9800255409045097
 1.0
 1.0

julia> constructRadauIIA5()
RadauIIA5{implicit, stages=3, order=5}:
A:
 0.19681547722366044  -0.06553542585019836   0.023770974348220147
 0.39442431473908723   0.29207341166522843  -0.04154875212599793
 0.37640306270046725   0.5124858261884215    0.1111111111111111
b:
 0.37640306270046725  0.5124858261884215  0.1111111111111111
c:
 0.15505102572168222
 0.6449489742783179
 1.0
```

```julia
using DiffEqBase: DiffEqBase, RKTableau
function constructTsitouras5(T::Type = Float64)
  A = zeros(T,7,7)
  c = zeros(T,7)
  α = zeros(T,7)
  αEEst = zeros(T,7)
  c[2] =       convert(T,161//1000)
  c[3] =       convert(T,327//1000)
  c[4] =       convert(T,9//10)
  c[5] =       convert(T,big".9800255409045096857298102862870245954942137979563024768854764293221195950761080302604")
  c[6] =       convert(T,1)
  c[7] =       convert(T,1)
  A[2,1]  =       convert(T,161//1000)
  A[3,1]  =       convert(T,big"-.8480655492356988544426874250230774675121177393430391537369234245294192976164141156943e-2")
  A[3,2]  =       convert(T,big".3354806554923569885444268742502307746751211773934303915373692342452941929761641411569")
  A[4,1]  =       convert(T,big"2.897153057105493432130432594192938764924887287701866490314866693455023795137503079289")
  A[4,2]  =       convert(T,big"-6.359448489975074843148159912383825625952700647415626703305928850207288721235210244366")
  A[4,3]  =       convert(T,big"4.362295432869581411017727318190886861027813359713760212991062156752264926097707165077")
  A[5,1]  =       convert(T,big"5.325864828439256604428877920840511317836476253097040101202360397727981648835607691791")
  A[5,2]  =       convert(T,big"-11.74888356406282787774717033978577296188744178259862899288666928009020615663593781589")
  A[5,3]  =       convert(T,big"7.495539342889836208304604784564358155658679161518186721010132816213648793440552049753")
  A[5,4]  =       convert(T,big"-.9249506636175524925650207933207191611349983406029535244034750452930469056411389539635e-1")
  A[6,1]  =       convert(T,big"5.861455442946420028659251486982647890394337666164814434818157239052507339770711679748")
  A[6,2]  =       convert(T,big"-12.92096931784710929170611868178335939541780751955743459166312250439928519268343184452")
  A[6,3]  =       convert(T,big"8.159367898576158643180400794539253485181918321135053305748355423955009222648673734986")
  A[6,4]  =       convert(T,big"-.7158497328140099722453054252582973869127213147363544882721139659546372402303777878835e-1")
  A[6,5]  =       convert(T,big"-.2826905039406838290900305721271224146717633626879770007617876201276764571291579142206e-1")
  A[7,1]  =       convert(T,big".9646076681806522951816731316512876333711995238157997181903319145764851595234062815396e-1")
  A[7,2]  =       convert(T,1//100)
  A[7,3]  =       convert(T,big".4798896504144995747752495322905965199130404621990332488332634944254542060153074523509")
  A[7,4]  =       convert(T,big"1.379008574103741893192274821856872770756462643091360525934940067397245698027561293331")
  A[7,5]  =       convert(T,big"-3.290069515436080679901047585711363850115683290894936158531296799594813811049925401677")
  A[7,6]  =       convert(T,big"2.324710524099773982415355918398765796109060233222962411944060046314465391054716027841")
  α[1]    =       convert(T,big".9646076681806522951816731316512876333711995238157997181903319145764851595234062815396e-1")
  α[2]    =       convert(T,1//100)
  α[3]    =       convert(T,big".4798896504144995747752495322905965199130404621990332488332634944254542060153074523509")
  α[4]    =       convert(T,big"1.379008574103741893192274821856872770756462643091360525934940067397245698027561293331")
  α[5]    =       convert(T,big"-3.290069515436080679901047585711363850115683290894936158531296799594813811049925401677")
  α[6]    =       convert(T,big"2.324710524099773982415355918398765796109060233222962411944060046314465391054716027841")
  αEEst[1]=       convert(T,big".9468075576583945807478876255758922856117527357724631226139574065785592789071067303271e-1")
  αEEst[2]=       convert(T,big".9183565540343253096776363936645313759813746240984095238905939532922955247253608687270e-2")
  αEEst[3]=       convert(T,big".4877705284247615707855642599631228241516691959761363774365216240304071651579571959813")
  αEEst[4]=       convert(T,big"1.234297566930478985655109673884237654035539930748192848315425833500484878378061439761")
  αEEst[5]=       convert(T,big"-2.707712349983525454881109975059321670689605166938197378763992255714444407154902012702")
  αEEst[6]=       convert(T,big"1.866628418170587035753719399566211498666255505244122593996591602841258328965767580089")
  αEEst[7]=       convert(T,1//66)
  A = map(T,A)
  α = map(T,α)
  c = map(T,c)
  αEEst = map(T,αEEst)
  return(RKTableau(A,c,α,5,bEEst=αEEst,adaptiveorder=4,name=:Tsitouras5))
end
function constructRadauIIA5(T::Type = Float64)
  sq6 = sqrt(6)
  A = [11//45-7sq6/360 37//225-169sq6/1800 -2//225+sq6/75
       37//225+169sq6/1800 11//45+7sq6/360 -2//225-sq6/75
       4//9-sq6/36 4//9+sq6/36 1//9]
  c = [2//5-sq6/10;2/5+sq6/10;1]
  α = [4//9-sq6/36;4//9+sq6/36;1//9]
  A = map(T,A)
  α = map(T,α)
  c = map(T,c)
  return RKTableau(A,c,α,5,name=:RadauIIA5)
end
```

**Comment by ChrisRackauckas** [comment]
why does it need to be generated?

**Comment by ChrisRackauckas** [comment]
Can it be done in steps? New format, solver updates, devtools updates, old format deprecated?

**Comment by YingboMa** [comment]
Because it automatically detects the explicitness and FSAL on construction and put the result in the type. We will need `@generated` either way when we use `BigFloat` inside.

File: src/tableaus.jl [hunk 1]
```diff
@@ -1,37 +1,153 @@
 """
-$(TYPEDEF)
+  $(TYPEDEF)
```
Comment (devmotion) [inline src/tableaus.jl:2]: I think this should just be
```suggestion
$(TYPEDEF)
```

**Comment by ChrisRackauckas** [comment]
What's the state of this?

**Pull request closed without merging**